Task2.py

```diff
@@ -0,0 +1,14 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+
+# Пример:
+
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+a = int(input("Введите первую цифру: "))
+b = int(input("Введите вторую цифру: "))
+c = int(input("Введите третью цифру: "))
+sum = (a+b+c)
+print("Сумма чисел")
+print(a,b,c,'=', sum)
```

Comment:
Не забывайте ставить пробелы.
```suggestion
print(a, b, c, '=', sum)
```